source/release-notes.txt

```diff
@@ -10,6 +10,43 @@ Release Notes
    :depth: 1
    :class: twocols
 
+1.3.4 Changelog
+---------------
+
+*Released December 11, 2023*
+
+Improvements:
+
+- Improved SQL query conversion quality for the Query Converter 
+  (private preview). For details on how to join the Query Converter 
+  early access program, see :ref:`rm-query-converter`.
+- Improved application performance through expanded use of available 
+  memory in suitable server environments.
+
+New features:
+
+- `Sybase ASE database 
```

Comment:
Should we add a footnote to the https://www.mongodb.com/docs/relational-migrator/supported-databases/ page indicating the limited support functionality added in this release? It seems odd it is mentioned in RN's but not mentioned on that page.

Reply:
**Disregard**

This is covered in the recently filed ticket: https://jira.mongodb.org/browse/DOCSP-34984